website/docs/getting-started/cluster-setup/kubernetes.md

```diff
@@ -1,8 +1,101 @@
 ---
 title: Kubernetes
-sidebar_position: 2
+sidebar_position: 1
 ---
 
 # Setup a minikube cluster
+Please note that this documentation is only useful in setting up a development environment, it is not recommended for a production environment.
 
-Please make sure you have read [this document](development-clusters.md) first.
\ No newline at end of file
+This guide assumes that you have [installed minikube](https://minikube.sigs.k8s.io/docs/start/) on your system.
+
+If you are using a Kubernetes cluster other than minikube, this guide assumes that you have admin privileges to the cluster and are logged in with the admin user; operator installation is only possible with an admin user.
+
+**Agenda for this guide:**
+* [Install the OLM](#install-the-olm)
+* [Install the Service Binding Operator](#install-the-service-binding-operator)
+* [Install an operator](#install-an-operator)
+* [Verify the operator installation](#verify-the-operator-installation)
+
```

Comment:
TODO: Add a note about enabling Ingress and Ingress-dns addon which is required for URL to work on minikube and perhaps kubernetes cluster.